docs/hikey.md

```diff
@@ -57,6 +57,9 @@ before building, see [UART in hikey.mk].
 Just follow the "Get and build the solution" in the [README.md] file. The `make
 flash` step will tell you how you should set the jumpers on the board.
 
+**NOTE:** If you're using the Kirin 620 or Hikey from CircuitCo, please set
```

Comment:
I was writing based on https://github.com/OP-TEE/build/blob/master/docs/hikey.md#3-supported-hikey-boards, which seems to be using the term 'Kirin 620' to refer to the 1st gen hikey boards with 4gb and uart0 soldered. Should I just change it to 'If you're using a Hikey board with a 4GB flash, please set..' instead?